server/gaps/items/gaps_code_creator.py

```diff
@@ -0,0 +1,108 @@
+import csv
```

Comment:
As discussed, a comment up the top here to explain the purpose/status of this script would be good